document/Conventions.mdk

```diff
@@ -4,6 +4,11 @@
 
 The following conventions are used within this document.
 
+## Key words for requirements levels
+
+In this document, the key words "must", "must not", "required", "shall", "shall not",
+"should", "should not", "recommended", "may", and "optional" are used as defined in RFC 2119.
```

Comment:
There aren't any inline links from the spec's text to the references. All the links are in the Normative References section.